react/features/invite/functions.ts

```diff
@@ -534,25 +571,37 @@ function isPhoneNumberRegex(): RegExp {
  * @param {string} jwt - The jwt token to pass to the search service.
  * @param {string} text - Text to search.
  * @param {Array<string>} queryTypes - Array with the query types that will be
- * executed - "conferenceRooms" | "user" | "room".
+ * executed - "conferenceRooms" | "user" | "room" | "email".
  * @returns {Promise} - The promise created by the request.
  */
 export function searchDirectory( // eslint-disable-line max-params
         serviceUrl: string,
         jwt: string,
         text: string,
-        queryTypes: Array<string> = [ 'conferenceRooms', 'user', 'room' ]
+        queryTypes: Array<string> = [ 'conferenceRooms', 'user', 'room', 'email' ],
+        peopleSearchTokenLocation?: string,
+        peopleSearchTokenKey?: string,
 ): Promise<Array<{ type: string; }>> {
 
     const query = encodeURIComponent(text);
     const queryTypesString = encodeURIComponent(JSON.stringify(queryTypes));
 
+    // Parse all the query strings of the search directory endpoint
+    const params = new URLSearchParams();
+    params.append('query', query);
+    params.append('queryTypes', queryTypesString);
+    params.append('jwt', jwt);
+
+    if (peopleSearchTokenLocation && peopleSearchTokenKey) {
+        const peopleSearchToken = localStorage.getItem(peopleSearchTokenLocation) ?? "";
+        params.append(peopleSearchTokenKey, peopleSearchToken);
```

Comment:
Same here why don't you use Authorization': `Bearer?

Reply:
Thank you for the quick review.

For first I will check all the linter errors today and make sure `npm run lint` passes.

As for the JWT in the headers, it was not there three months ago when I first did the PR. I just noticed the change today when remerging master to the branch, so I will update things consequently ;-)

I'll put a comment when everything will be settled.